домашнее2/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace домашнее2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+
+            Console.Write("Введите длину стороны квадрата: ");
+            double side = Convert.ToDouble(Console.ReadLine());
```

Comment:
Не стоит использовать методы класса Convert без особых причин. Лучше так

```suggestion
            double side = double.Parse(Console.ReadLine());
```